modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -11,11 +11,11 @@ This procedure relies on the newly introduced Shared Resources CSI Driver featur
 [IMPORTANT]
 ====
 [subs="attributes+"]
-The Shared Resources CSI Driver and The Build CSI Volumes are both Technology Preview features, which are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+The Shared Resources CSI Driver is a Technology Preview feature, which is not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using this in production. This feature provides early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
 
 For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
 
-The Shared Resources CSI Driver and the Build CSI Volumes features also belong to the `TechPreviewNoUpgrade` feature set, which is a subset of the current Technology Preview features. You can enable the `TechPreviewNoUpgrade` feature set on test clusters, where you can fully test them while leaving the features disabled on production clusters. Enabling this feature set cannot be undone and prevents minor version updates. This feature set is not recommended on production clusters. See "Enabling Technology Preview features using feature gates" in the following "Additional resources" section.
+The Shared Resources CSI Driver feature also belongs to the `TechPreviewNoUpgrade` feature set, which is a subset of the current Technology Preview features. You can enable the `TechPreviewNoUpgrade` feature set on test clusters, where you can fully test them while leaving the features disabled on production clusters. Enabling this feature set cannot be undone and prevents minor version updates. This feature set is not recommended on production clusters. See "Enabling Technology Preview features using feature gates" in the following "Additional resources" section.
 ====
```

Comment:
I can see why the TP snippet was not used at first, since it was trying to cover two features and therefore plural was needed. However, now that this is a single feature, it would be better to revert to the standard way of doing this. The feature gate explanation should be fine outside of an admonition, so I would suggest this:

```
:FeatureName: The Shared Resources CSI Driver
include::snippets/technology-preview.adoc[]

The Shared Resources CSI Driver feature also belongs to the `TechPreviewNoUpgrade` feature set, which is a subset of the current Technology Preview features. You can enable the `TechPreviewNoUpgrade` feature set on test clusters, where you can fully test them while leaving the features disabled on production clusters. Enabling this feature set cannot be undone and prevents minor version updates. This feature set is not recommended on production clusters. See "Enabling Technology Preview features using feature gates" in the following "Additional resources" section.
```